src/components/atoms/Checkbox/Checkbox.styles.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+
+import { CheckboxStateColor } from '@src/types/component'
+
+export const CheckboxWrapper = styled.div<CheckboxStateColor>`
```

Comment:
I thought we don't use styled-component for exported project? 🤔

Reply:
Ah right, initially, I saw @duongjitera use styled component for Visibility thing, I just imitated it 🤔